### PR TITLE
Update for versioningit and stuff

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

**Comment by IAlibay** [comment]
CI  hasn't been running for a long while, let's see what happens...

**Comment by orbeckst** [comment]
Thanks for starting the work.

For reference I am copying the current failures from setuptools:
```
[WARNING ] versioningit: Version extracted from tag 'release-2.0.0' is not PEP 440-compliant
      [ERROR   ] versioningit: InvalidVersionError: Cannot parse version 'release-2.0.0'
      error: Multiple top-level packages discovered in a flat-layout: ['devtools', 'propkatraj'].
      
      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.
      
      If you are trying to create a single distribution with multiple packages
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:
      
      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names
```

1. need to change how we tag to make versioningit happy
2. need to fix something related to the package layout

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+"source" = "https://github.com/Becksteinlab/propkatraj"
+
+[tool.versioningit]
+default-version = "1+unknown"
+
+[tool.versioningit.vcs]
+method = "git"
+# the below line expects tags to look like '1.0.2'.
+# if prefixing with a v, e.g. 'v1.0.2', change it to ["v*"]
+match = ["release-*"]
```
Comment (orbeckst) [inline pyproject.toml:77]: I am now surprised that versioningit failed, given that it should have found the tag. Maybe not PEP440 compliant?

File: setup.py
Comment (orbeckst) [inline setup.py:1]: hooray!

Comment (orbeckst) [reply setup.py:1]: setup.py gone!

**Comment by IAlibay** [comment]
@orbeckst I think this is good to go?

File: propkatraj/tests/data/.atterr_frame1.xtc_offsets.lock
Comment (orbeckst) [inline propkatraj/tests/data/.atterr_frame1.xtc_offsets.lock:1]: Should this be really included?

File: propkatraj/tests/data/.indexerr_frame14.xtc_offsets.lock
Comment (orbeckst) [inline propkatraj/tests/data/.indexerr_frame14.xtc_offsets.lock:1]: Should this included?

**Review by orbeckst — CHANGES_REQUESTED** [review]
Many thanks. There are two empty offset lock files checked in. Should they be included?

File: propkatraj/tests/data/.atterr_frame1.xtc_offsets.lock
Comment (IAlibay) [reply propkatraj/tests/data/.atterr_frame1.xtc_offsets.lock:1]: uh oh, looks like something went wrong with the gitignore!

**Comment by IAlibay** [comment]
This should be good to go for now @orbeckst - next up let's aim for the latest propka release!

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Cheers and yes!

**Pull request merged**